src/OrchardCore.Modules/OrchardCore.ContentPreview/Views/Preview/Index.cshtml

```diff
@@ -111,8 +111,8 @@
                     if (iframe && iframe.contentWindow) {
                         iframe.contentWindow.document.open();
                         iframe.contentWindow.document.write(data);
-                        iframe.contentWindow.document.close();
                         previewRenderData = data;
+                        iframe.contentWindow.document.close();
```

Comment:
Could you please provide animated GIF coz Seb always like screenshots & demos ;)

Reply:
While trying to get Screenshots my Chromium browsers updated to Chromium 98.0.1089.1 and the issue no longer occurs. Since it seems Chromium is rolling out the fix on their end I suppose we can cancel this PR and close the issue.

Sorry for the hassle.

Reply:
Thanks a lot of let us know that the update fix the issue

Reply:
@Ranger1230 This issue isn't resolved on macOS  in the latest Safari. Why not import the PR into the main branch?

Saferi: 15.4
Latest Monterey: ver 12.3.1
CPU Intel i9.